acquisitions/src/test/java/org/folio/OrdersApiTest.java

```diff
@@ -178,6 +178,11 @@ void shouldDecreaseQuantityWhenDeletePieceWithNoLocation() {
     runFeatureTest("should-decrease-quantity-when-delete-piece-with-no-location");
   }
 
+  @Test
+  void testCrossTenantInventoryObjectsCreationWhenCreatingPiece() {
+    runFeatureTest("pieces-api-test-for-cross-tenant-envs");
+  }
+
```

Comment:
check one more time, I believe you don't need to OdersApiTest add feature that belong to consortia

Reply:
Yes correct, left that one from an old commit